extensions/validation/src/test/java/org/mule/extension/validation/BasicValidationTestCase.java

```diff
@@ -149,22 +156,17 @@ public void isFalse() throws Exception
     public void notEmpty() throws Exception
     {
         final String flowName = "notEmpty";
-
```

Comment:
we use empty lines to separate sections on our tests and make it more readable all the time. Actually, the original PR passed code review in that form. If you check other test cases of other modules you'll find that all over the place